_episodes_rmd/11-supp-read-write-csv.Rmd

```diff
@@ -115,6 +115,19 @@ carSpeeds$Color
 ```
 
 That's better! And we can see how the data now is read as character instead of factor.
+ 
+ 
+ 
+Alternatively, if we want to prevent characters from converting to factors, we can change R's default behavior by using `options(stringsAsFactors=FALSE)`. This will change the default value of the 'stringsAsFactors' argument to FALSE in all functions that utilize this parameter without us having to do so manually. 
+
+```{r optionsStringsAsFactorFALSE}
+options(stringsAsFactors=FALSE)
+carSpeeds <- read.csv(file = 'data/car-speeds.csv')
+str(carSpeeds)
+```
+
+See ?options for more details about the function and a list of other default settings that can be changed.
```

Comment:
Would this be a good place to mention that each `options()` change is valid only during the current R session?

Since this touches on using [`.Rrofile` to change the defaults permanently](https://www.r-bloggers.com/fun-with-rprofile-and-customizing-r-startup/), and on rather teaching the tidyverse way (see #276), I'm worried that this change risks bloating the lesson.

Reply:
Agreed. I might even suggest a change above, to line 121, and abandon this part:

>If we want to prevent characters from converting to factors, we can change R's behavior in the current session by using `options(stringsAsFactors = FALSE)`. See `?options` for more details about the function. Note that any changes made using `options` will not be reflected in a new R session, unless the command is re-run.